versioned_docs/version-3.0.0-beta/quick-start/minting-101.md

```diff
@@ -0,0 +1,56 @@
+---
+id: minting-101
+title: Minting 101
+sidebar_label: Minting 101
+slug: /quick-start/minting-101
+---
+
+Hey there, in this tutorial we would show you how to mint your first  NFT on Spheron after you have been whitelisted.
+
+Note: This tutorial is for users that are accessing Spheron for the first time.
+
+
+### Step 1: Click the Find Out button
+On the home page of Spheron v3 as displayed below, click on the **Find Out** button to confirm if you have been whitelisted.
+
+<img src={require('@site/static/img/3.0.0/step1.png').default} /> <br/><br/>
+
+### Step 2: Connect your wallet
+Clicking on the **find out** button asks you to connect your wallet address (metamask or wallet connect) to spheron as displayed below.
+
+<img src={require('@site/static/img/3.0.0/step2a.png').default} /> <br/><br/>
+
+#### **Click on next**
+
+<img src={require('@site/static/img/3.0.0/step2b.png').default} /> <br/><br/>
+
+#### **Click on the connect button**
+
+### Step 3: Minting an NFT
+
+<img src={require('@site/static/img/3.0.0/step3a.png').default} /> <br/><br/>
+
+As shown above, click on the **Mint A Key** button,and it pops up your wallet notification to say confirm the transaction.
+
+<img src={require('@site/static/img/3.0.0/step3b.png').default} /> <br/><br/>
+
+### Step 4: Getting your tokens
+If you got the insufficient funds notification as shown above to mint your NFT, Click [here](https://faucet.polygon.technology/) and paste your wallet address in the wallet address Box as shown below, then click submit.
+<img src={require('@site/static/img/3.0.0/step4a.png').default} /> <br/><br/>
+
+#### CLICK THE CONFIRM BUTTON TO CONFIRM TRANSACTION AND WAIT FOR TOKENS TO REFLECT IN YOUR WALLET
+<img src={require('@site/static/img/3.0.0/step4b.png').default} /> <br/><br/>
+
+### Final step
+When the tokens have been reflected  in your wallet, go back to your NFT minting Page and click on **MINT A KEY** as shown in [step3](#step-3-minting-an-nft). Confirm the transaction as seen below
+<img src={require('@site/static/img/3.0.0/finalb.png').default} /> <br/><br/>
+
+***Wait for the loader to load...***
+
+<img src={require('@site/static/img/3.0.0/loader.png').default} /> <br/><br/>
+
+### voila!! You have just minted your NFT
```

Comment:
put another note here: if you see an error popping in this step, please refresh the window and just click on find out again, you will be able to view your NFTs and launch the app